docs/src/guide/processes.rst

```diff
@@ -126,7 +128,8 @@ of ``uv_kill`` is::
 
 For processes started using libuv, you may use ``uv_process_kill`` instead,
 which accepts the ``uv_process_t`` watcher as the first argument, rather than
-the pid. In this case, **remember to call** ``uv_close`` on the watcher.
+the pid. In this case, **remember to call** ``uv_close`` on the watcher after
+the execution of the exit callback.
```

Comment:
```suggestion
the pid. In this case, **remember to call** ``uv_close`` on the watcher _after_
the exit callback has been called.
```
Also just a suggestion but libuv's docs pretty much always say callbacks are called, not executed.